flink-end-to-end-tests/test-scripts/common_mesos_docker.sh

```diff
@@ -53,11 +55,22 @@ function start_flink_cluster_with_mesos() {
     set_config_key "jobmanager.rpc.address" "mesos-master"
     set_config_key "rest.address" "mesos-master"
 
-    docker exec -itd mesos-master bash -c "${FLINK_DIR}/bin/mesos-appmaster.sh -Dmesos.master=mesos-master:5050"
+    docker cp ${FLINK_DIR} mesos-master:$MESOS_FLINK_DIR
```

Comment:
Why not bundle Flink and the test data inside the image?

Reply:
> all writes should go to a volume or bind mount

Regarding this concern, I think that the two approaches do not have significant difference. Since We'll always write the log to the container's storage. If you have a strong point here, I'm ok to bundle Flink and the test data inside the image.

Reply:
Anyways, if we can use _bind mounts_ again, this isn't needed.